modules/rotating-certificate-authority

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// security/certificate_types_descriptions/etcd-certificates.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rotating-certificate-authority_{context}"]
+=== Rotating the etcd certificate authority
+
+Rotate the `etcd` certificate authority (CA) before the certificate expires.
+
+.Procedure
+
+. Verify the remaining lifetime of the new signer certificate:
+
```

Comment:
@tmalove Try adding a `+` here and see if that helps.

Reply:
@lahinson the problem was the file suffix...it is corrected now, thanks!